File: 1332-remove-palindromic-subsequences/1332-remove-palindromic-subsequences.java
class Solution {
    public int removePalindromeSub(String s) {
        char[] x = s.toCharArray();
        int n = x.length;
        int i = 0;
        int j = n-1;
        
        while(i<j){
            if(x[i] != x[j]){
                return 2;
            }
            i++;
            j--;
        }
        return 1;
    }
}

File: 1332-remove-palindromic-subsequences/1332-remove-palindromic-subsequences.py
class Solution(object):
    def removePalindromeSub(self, s):
        """
        :type s: str
        :rtype: int
        """
        n = len(s)
        
        i = 0
        j = n-1
            
        while(i < j):
            if(s[i] != s[j]):
                return 2
            else:
                i += 1
                j -= 1
            
        return 1